ayon_bin_distro/gui/progress_ui.py

```diff
@@ -146,17 +179,28 @@ def _close_window(self):
 class ProgressDialog(QtWidgets.QDialog):
     def __init__(
         self,
-        controller,
+        controller: worker.Controller,
         delet_progress_bar_on_finish: bool = True,
         close_on_finish: bool = False,
         auto_close_timeout: float = 0.5,
         title: str = "",
+        ui_update_interval: int = 100,
     ):
+        """Class to display the progress of the worker.Controller to a user.
+
+        Args:
+            controller: instance of worker.Controller that should be monitored
+            delet_progress_bar_on_finish: bool to allow for progress bars to be deleted when they finished
+            close_on_finish: bool to decided if the Gui needs to be closed manually
+            auto_close_timeout: float timeout value for how long the GUI should exist after all process bars have finished
+            title: str GUI window title
+            ui_update_reate: defines the interval that is used to ask the worker.Controller for the current state of the tasks
```

Comment:
Note the argument typo fix from `delet_progress_bar_on_finish` to `delete_progress_bar_on_finish`.

```suggestion
        Args:
            controller (worker.Controller): controller that should be monitored
            delete_progress_bar_on_finish (bool) to allow for progress bars to be deleted when they finished
            close_on_finish (bool): to decided if the Gui needs to be closed manually
            auto_close_timeout (float): timeout value for how long the GUI should exist after all process bars have finished
            title (str): GUI window title
            ui_update_interval (int): defines the interval that is used to ask the `worker.Controller` for the current state of the tasks
```

Reply:
done in external commit.

Reply:
I guess this is outdated now - the typo I believe is gone, maybe the type hints are not there yet... But I'm getting fed up with how Github shows the changes and hides half of it all the time :')